packages/beacon-node/test/sim/mergemock.test.ts

```diff
@@ -159,7 +172,7 @@ describe("executionEngine / ExecutionEngineHttp", function () {
       defaultConfig: {
         graffiti: "default graffiti",
         strictFeeRecipientCheck: true,
-        feeRecipient: "0xcccccccccccccccccccccccccccccccccccccccc",
+        feeRecipient: "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
```

Comment:
Store this addresses in variables to link param setting and assertion. With names like: feeRecipientHappyCase, feeRecipientLocal, feeRecipientMevBoost, etc

Reply:
sounds good updated :+1: